.github/workflows/flowzone.yml

```diff
@@ -6,12 +6,21 @@ on:
     branches:
       - "main"
       - "master"
+  # allow external contributions to use secrets within trusted code
+  pull_request_target:
+    types: [opened, synchronize, closed]
+    branches: [main, master]
 
 jobs:
   flowzone:
     name: Flowzone
-    uses: product-os/flowzone/.github/workflows/flowzone.yml@vipul/custom-build
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
```

Comment:
Could you rebase the 2nd commit to include the additional changes to this file?

Reply:
Have never tried to do it but on the interactive rebase I used the e option to edit the commit and it end up swallowing the commit entirely. I only have 5 commits locally for some reason.

Reply:
Will pull and try again.

Reply:
Edited the 2nd commit as mentioned.

Reply:
You sure? The second commit still shows the old values. I would prefer this commit not contain any changes to the workflow and there be a separate commit for the Flowzone changes only.

Reply:
The second commit has changes that make sense for the commit. It's commit for Deploying to CF. I added configuration changes needed to deploy to CF? Not sure why there should be a separate commit for Flowzone changes. Here's what it shows me.

![Screenshot from 2022-12-29 17-10-41](https://user-images.githubusercontent.com/22801822/209946193-877e1327-80e6-41bc-bd64-982dd18c8282.png)